miner/payload_building_test.go

```diff
@@ -331,14 +372,23 @@ func genTxs(startNonce, count uint64) types.Transactions {
 	txs := make(types.Transactions, 0, count)
 	signer := types.LatestSigner(params.TestChainConfig)
 	for nonce := startNonce; nonce < startNonce+count; nonce++ {
-		txs = append(txs, types.MustSignNewTx(testBankKey, signer, &types.AccessListTx{
+		// generate incompressible data to put in the tx for DA filter testing. each of these
+		// txs will be bigger than the 100 minimum.
+		randomBytes := make([]byte, 100)
+		_, err := rand.Read(randomBytes)
+		if err != nil {
+			panic(err)
```

Comment:
`t.Fatal` or `t.Fatalf` is preferable to a raw panic in tests

Reply:
I generally prefer using panic() for "this should never happen" kinds of scenarios like this to distinguish them from actual test failures.

Reply:
Agree here, this would be a problem with the Go runtime.